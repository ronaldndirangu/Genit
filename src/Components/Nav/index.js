import React from "react";
import "./index.scss";

const Nav = ({ brand }) => {
  return <div className="nav-brand">{brand}</div>;
};

export default Nav;
