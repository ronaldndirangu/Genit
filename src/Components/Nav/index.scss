.nav-brand {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 4rem;
  font-size: 2.5rem;
  border-bottom: solid 1px #ddd;
  font-family: monospace;
  color: #39298c;
  box-shadow: 2px 2px 2px #ccc;
}
