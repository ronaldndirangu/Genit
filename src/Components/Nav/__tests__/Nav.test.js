import React from "react";
import { shallow } from "enzyme";
import Nav from "../../Nav";

describe("Nav", () => {
  it("Nav renders correctly", () => {
    const component = shallow(<Nav />);
    expect(component).toMatchSnapshot();
  });
});
