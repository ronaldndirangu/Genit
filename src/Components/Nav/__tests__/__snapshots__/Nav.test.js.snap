// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Nav renders correctly 1`] = `ShallowWrapper {}`;
